tests/many_to_many/tests.py

```diff
@@ -332,6 +332,45 @@ def test_remove(self):
             ])
         self.assertQuerysetEqual(self.a3.publications.all(), [])
 
+    def test_set(self):
+        self.p2.article_set.set(self.a4, self.a3)
+        self.assertQuerysetEqual(self.p2.article_set.all(),
+            [
```

Comment:
I don't like these either since they aren't pep8 compliant (E128) but the whole file (and the similar test packages) use them, so I'd like to keep them consistent.